srcpkgs/MultiMC/template

```diff
@@ -18,7 +18,7 @@ homepage="http://multimc.org"
 distfiles="https://github.com/${pkgname}/${pkgname}5/archive/${version}.tar.gz
```

Comment:
https://github.com/MultiMC/Launcher/archive/${version}.tar.gz

Reply:
I thought I had already done that. It should be good now.